.travis.yml

```diff
@@ -52,7 +52,7 @@ before_install:
 
 script:
   - if [[ "$JOB" == "check_style" ]]; then ./scripts/check_style.sh; fi
-  - if [[ "$JOB" == "test" ]]; then ./scripts/tests.sh all; WITH_PYTHON3=ON ./scripts/tests.sh all; fi
```

Comment:
Disable Py3 test for now as Travis CI is having problem installing 'six' package for Py2Py3 compatibility.